signer/src/wsts_state_machine.rs

```diff
@@ -465,41 +485,108 @@ impl SignerStateMachine {
             return Err(error::Error::InvalidConfiguration);
         };
 
-        let state_machine = WstsSigner::new(
+        let inner = WstsSigner::new(
             threshold,
             dkg_threshold,
             num_parties,
             num_keys,
             id,
             key_ids,
-            signer_private_key.into(),
+            private_key.into(),
             public_keys,
-            &mut rng,
+            &mut OsRng,
```

Comment:
Do we want use some `ChaCha20Rng` instead `OsRng` as well?

Reply:
Yea, I was thinking that rust don't guarantee that `OsRng` is CSPRNG, but just double checked and it seems that rust guarantee it.